web/styles/left_sidebar.css

```diff
@@ -401,6 +401,13 @@
 #stream_filters .narrow-filter.highlighted_stream {
     &.active-filter > .bottom_left_row {
         background-color: var(--color-background-hover-narrow-filter);
+        outline: 2px solid var(--color-outline-focus);
```

Comment:
The duplication here makes me a little uneasy. I'm also trying to understand why these are being declared outside of a pseudo-class like `:focus-within`. Do the classes mentioned here get added and removed as one tabs through?

Reply:
Okay. I guess this is set in the second commit. But I do wonder if it's possible to use a selector that handles all focus cases, rather than having to do this duplication.

Reply:
> The duplication here makes me a little uneasy. I'm also trying to understand why these are being declared outside of a pseudo-class like `:focus-visible`. Do the classes mentioned here get added and removed as one tabs through?

Ok, so the reason pseudo-classes like `:focus-visible` won't work here is due to the fact that we don't actually focus on these elements ever and just add a highlight class dynamically. The actual browser focus is always kept on the stream or user search bar upon hitting `q` or `w`, so that the user can search anytime even when navigating in the lists.

Reply:
But yeah we still didn't need that duplication, thanks for catching it, I must have left it there during debugging.